integration-tests/constants/dynamoDBTicketDetails.ts

```diff
@@ -0,0 +1,302 @@
+export const dynamoDBTicketDetails = {
+  ticket: {
+    id: 1520,
+    external_id: null,
+    via: {
+      channel: 'api',
+      source: { from: {}, to: {}, rel: null }
+    },
+    created_at: '2022-10-14T16:02:02Z',
+    updated_at: '2022-10-14T16:02:17Z',
+    type: null,
+    subject: 'Integration Test Request - 48',
+    raw_subject: 'Integration Test Request - 48',
+    description: 'PII request created in integration test',
+    priority: null,
+    status: 'closed',
+    recipient: null,
+    requester_id: 5647117262620,
+    submitter_id: 5647117262620,
+    assignee_id: null,
+    organization_id: null,
+    group_id: 5606344981660,
+    collaborator_ids: [],
+    follower_ids: [],
+    email_cc_ids: [],
+    forum_topic_id: null,
+    problem_id: null,
+    has_incidents: false,
+    is_public: true,
+    due_at: null,
+    tags: [
+      'approved',
+      'drivers_license',
+      'event_id',
+      'pii_ticf_analyst',
+      'ticf_analyst'
+    ],
+    custom_fields: [
+      { id: 360021265420, value: null },
```

Comment:
I presume the test would work if you left all these null fields out? If so, I would do that, because it's just wasted space and makes things harder to read.